packages/core/src/controllers/ApiController.ts

```diff
@@ -18,7 +18,7 @@ import { OptionsController } from './OptionsController';
 // -- Helpers ------------------------------------------- //
 const baseUrl = CoreHelperUtil.getApiUrl();
 const api = new FetchUtil({ baseUrl });
-const entries = '100';
+const defaultEntries = '100';
```

Comment:
Now that we have separate endpoint to fetch data about deeplinks on ios / android. Do pages need to be this big, perhaps something like `32` / `42` would be better?

Reply:
I use 48 indeed

Reply:
done